features/customizable-select.yml

```diff
@@ -4,3 +4,9 @@ spec: https://open-ui.org/components/customizableselect/
 group:
   - forms
   - html-elements
+compat_features:
```

Comment:
These are new, and come from https://github.com/mdn/browser-compat-data/pull/26171